.github/workflows/doc-deploy.yml

```diff
@@ -0,0 +1,52 @@
+name: doc-deploy
```

Comment:
Since there is no documentation build right now in the repo, you can remove this file ;)

Reply:
Done